test/functional/http/get/transactions.js

```diff
@@ -1040,6 +1049,54 @@ describe('GET /api/transactions', () => {
 			});
 		});
 
+		describe('signature', () => {
+			it('should not show signSignature when empty upon registering second passphrase', async () => {
+				// Prepare
+				const transaction = lisk.transaction.registerSecondPassphrase({
+					passphrase: accountSecondPass.passphrase,
+					secondPassphrase: accountSecondPass.secondPassphrase,
+				});
+
+				await apiHelpers.sendTransactionPromise(transaction, 200);
+				await waitFor.confirmations([transaction.id]);
+
+				// Act
+				const {
+					body: { data: transactions },
+				} = await transactionsEndpoint.makeRequest(
+					{
+						type: transactionTypes.SIGNATURE,
+						limit: 1,
+						senderPublicKey: accountSecondPass.senderId,
+						sort: 'timestamp:desc',
+					},
+					200
+				);
+				// Assert
+				expect(transactions[0]).to.not.have.property('signSignature');
+			});
+
+			it('should show signSignature whem signing a transfer transaction with second passphrase', async () => {
+				// Prepare
+				const transaction = lisk.transaction.transfer({
+					amount: 1,
+					passphrase: accountSecondPass.passphrase,
+					secondPassphrase: accountSecondPass.secondPassphrase,
+					recipientId: accountFixtures.existingDelegate.address,
+				});
+
+				await sendTransactionPromise(transaction, 200);
+				await waitFor.confirmations([transaction.id]);
+
+				// Act
+				const {
+					body: { data: transactions },
+				} = await transactionsEndpoint.makeRequest({ id: transaction.id }, 200);
+				// Assert
+				expect(transactions[0].signSignature).to.not.be.empty;
```

Comment:
this looks good, what would be nice to have is comparing `signSignature` against the expecting `signSignature`
something like `expect(transactions[0].signSignature).to.eq(signSignatureFuc(required params))`. but for now i am goona approve, you can use this pattern for future, this way the expectation is spot on.

Reply:
Ok, thanks!